modules/edit-modes/src/lib/draw-rectangle-mode.ts

```diff
@@ -1,10 +1,11 @@
 import bboxPolygon from '@turf/bbox-polygon';
+import rewind from '@turf/rewind';
 import { Position, Polygon, FeatureOf } from '../geojson-types';
 import { TwoClickPolygonMode } from './two-click-polygon-mode';
 
 export class DrawRectangleMode extends TwoClickPolygonMode {
   getTwoClickPolygon(coord1: Position, coord2: Position, modeConfig: any): FeatureOf<Polygon> {
-    const rectangle = bboxPolygon([coord1[0], coord1[1], coord2[0], coord2[1]]);
+    const rectangle = rewind(bboxPolygon([coord1[0], coord1[1], coord2[0], coord2[1]]));
```

Comment:
Can you make this more generic by adding the rewind call in `TwoClickPolygonMode` where it calls `getTwoClickPolygon`?

Reply:
I should be more explicit. Can you make this change in `TwoClickPolygonMode.handleClick` instead? 

this:
```
const editAction = this.getAddFeatureOrBooleanPolygonAction(feature, props);
```

can become
```
const editAction = this.getAddFeatureOrBooleanPolygonAction(rewind(feature), props);
```

Then every mode that extends `TwoClickPolygonMode` won't need to change.